frontend/www/js/omegaup/components/arena/NewClarificationPopup.vue

```diff
@@ -89,9 +80,11 @@ export default class ArenaNewClarificationPopup extends Vue {
   // when PR #5126 or #5131 are merged
   @Prop({ default: () => [] }) problems!: types.NavbarContestProblem[];
   @Prop({ default: () => [] }) users!: types.ContestUser[];
-  @Prop() newClarification!: NewClarification | NewAdminClarification;
+  @Prop({ default: null }) problemAlias!: string;
```

Comment:
```suggestion
  @Prop({ default: null }) problemAlias!: string | null;
```

es ilegal asignarle `null` por default a algo que no declara que puede ser nulo

Reply:
Listo